google/cloud/forseti/services/model/model.proto

```diff
@@ -59,8 +59,9 @@ message ModelSimplified {
   string handle = 1;
   string name = 2;
   string status = 3;
-  string description = 4;
-  string message = 5;
+  string createdAt = 4;
```

Comment:
Is it okay the change the id here?  IIUC, the ids should not be changed, but I'm not sure why that is.  Do you?

https://developers.google.com/protocol-buffers/docs/proto3

![image](https://user-images.githubusercontent.com/6977544/39554873-2e632a46-4e2a-11e8-86bd-f7f097da8239.png)

Reply:
Updated.